trainingaddon/gensrc/de/hybris/training/constants/GeneratedTrainingaddonConstants.java

```diff
@@ -1,7 +1,7 @@
 /*
```

Comment:
@amboraes1 please configure the gitignore file to avoid Generated* files are included in your Pull Request, I will declide this and create a new one without Generated files